server/controllers/prisonerCellHistoryController.ts

```diff
@@ -68,16 +68,15 @@ export default class PrisonerCellHistoryController {
     }
 
     try {
-      // Parse query params for paging, sorting and filtering data
-      const { clientToken } = res.locals
-
-      const { bookingId, firstName, middleNames, lastName } = prisonerData
+      const { bookingId, firstName, middleNames, lastName, prisonId } = prisonerData
       const name = formatName(firstName, middleNames, lastName, { style: NameFormatStyle.firstLast })
 
-      const prisonApiClient = this.prisonApiClientBuilder(clientToken)
+      const prisonApiClient = this.prisonApiClientBuilder(res.locals.clientToken)
 
-      const page = 0
-      const cells = await prisonApiClient.getOffenderCellHistory(bookingId, { page, size: 10000 })
+      const [cells, receptions] = await Promise.all([
+        prisonApiClient.getOffenderCellHistory(bookingId, { page: 0, size: 10000 }),
+        prisonApiClient.getReceptionsWithCapacity(prisonId),
```

Comment:
As part of the follow up work to refactor this code, I'm going to look at conditionally calling the receptionsWithCapacity API, if the button isn't going to be displayed, we don't need the extra API call